data_models/data_refinery_models/models/organism.py

```diff
@@ -0,0 +1,121 @@
+import requests
+from xml.etree import ElementTree
+from django.db import models
+from data_refinery_models.models.base_models import TimeTrackedModel
+
+# Import and set logger
+import logging
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+NCBI_ROOT_URL = "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/"
+ESEARCH_URL = NCBI_ROOT_URL + "esearch.fcgi"
+EFETCH_URL = NCBI_ROOT_URL + "efetch.fcgi"
```

Comment:
There was definitely a reason or two why I decided to go this route, however they happened in person. What I can think of now though is that the `django-organisms` model has two names, common and scientific. However the NCBI API has a list structure for names which can include more than two and I wanted to create a model which could support that. Ultimately I think that the API I was trying to create (`get_id_for_name` and `get_name_for_id`) was more important than the data model I was creating to support it.

As to your other point, I'm not sure I like the idea of allowing someone to override the URLs without modifying the source code because it seems highly unlikely that someone could just pop in a different URL and have it work. The parameters that used for each request would almost certainly need to change and if they didn't it could cause undefined behavior. What use case do you think might be supported by allowing the URLs to be via the `settings.py` file?